lib/proxy/log.rb

```diff
@@ -13,7 +20,18 @@ def logger
     def self.logger
       log_file = ::Proxy::SETTINGS.log_file
       if log_file.upcase == 'STDOUT'
+        if SETTINGS.daemon
+          puts "Settings log_file=STDOUT and daemon=true are incompatible, exiting..."
+          exit 1
+        end
         logger = ::Logger.new(STDOUT)
+      elsif log_file.upcase == 'SYSLOG'
```

Comment:
Should probably mention `SYSLOG` as a possible value in the example config file.